src/sql/platform/accounts/test/browser/accountDialogController.test.ts

```diff
@@ -86,7 +86,7 @@ function createInstantiationService(addAccountFailureEmitter?: Emitter<string>):
 		.returns(() => undefined);
 
 	// Create a mock account dialog
-	let accountDialog = new AccountDialog(null, null, instantiationService.object, null, null, null, null, new MockContextKeyService(), null, undefined);
+	let accountDialog = new AccountDialog(null, null, instantiationService.object, null, null, null, null, new MockContextKeyService(), null, undefined, undefined);
 	let mockAccountDialog = TypeMoq.Mock.ofInstance(accountDialog);
```

Comment:
huh, why do we even have tests here instead of the sqltest folder? It's fine for now - just want to know if anyone knows of a reason (beyond "that's just how someone did it originally")

Reply:
I moved tests to work with vscode's layering tactic from the sqltest folder to where they should be for layering to apply to them.